app/views/organizations/_side_menu.html.erb

```diff
@@ -9,7 +9,7 @@
                 class: "menu-item #{'selected' if current_page?(organization_student_identifier_types_path(@organization))}" %>
   <% end %>
 
-  <% if student_identifier_enabled? %>
+  <% if team_management_enabled? %>
```

Comment:
This is the fix, we are actually flipping the team management feature here. Sorry for the mistake.